monai/bundle/scripts.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional, Sequence, Union
+
+from monai.bundle.utils import id_value_str_to_dict, parse_config_file, update_default_args
+from monai.utils import optional_import
+
+fire, _ = optional_import("fire")
+
+
+def run(
+    meta_file: Optional[Union[str, Sequence[str]]] = None,
+    config_file: Optional[Union[str, Sequence[str]]] = None,
+    override: Optional[Union[Dict, str]] = None,
+    target: Optional[str] = None,
+    args_file: Optional[str] = None,
+):
+    """
+    Specify metadata file and config file to run a regular training or evaluation program.
```

Comment:
I think we should move the examples to a readme file in the folder monai/bundle, with more examples and an overview of the syntax @, #

Reply:
Yeah, we definitely need to prepare a detailed document for all the features in `monai.bundle`.
Maybe start from @ericspod 's PR: https://github.com/Project-MONAI/MONAI/pull/3834

Thanks.